cmd/deployment/create.go

```diff
@@ -36,18 +36,19 @@ var createCmd = &cobra.Command{
 	Use:     "create {--file | --size <int> --zones <string> | --topology-element <obj>}",
 	Short:   "Creates a deployment",
 	PreRunE: cobra.MaximumNArgs(0),
-	Long:    createLong,
-	Example: createExample,
```

Comment:
// nolint needs to be added to the definition of those constants so the linter passes when they're not used. Or we can simply comment the whole cost definition.